packages/oui-stepper/README.md

```diff
@@ -0,0 +1,251 @@
+        </oui-field>
+        <oui-field label="Dolor sit amet" size="xl">
+            <oui-textarea name="skippableTextarea" rows="7"
+                model="$ctrl.skippableTextarea">
+            </oui-textarea>
+        </oui-field>
+    </oui-step-form>
+</oui-stepper>
+```
+
+### Events on `oui-stepper`
+
+**Note**: If you want to access the forms inside `on-finish` callback, you need to use the `forms` variable as below.
+
+```html:preview
+<oui-stepper
+    on-init="$ctrl.onInit('Dolor sit amet !')"
+    on-finish="$ctrl.onFinish(forms)">
+    <oui-step-form
+        name="eventsStepper"
+        header="Events on oui-stepper">
+        <oui-field label="Lorem ipsum" size="xl">
+            <input class="oui-input" name="initStepper"
+                type="text" required
+                ng-model="$ctrl.onInitModel">
+        </oui-field>
+    </oui-step-form>
+</oui-stepper>
+<div class="oui-doc-preview-only">
+    <p>On Finish</p>
+    <pre>{{$ctrl.onFinishForms | json}}</pre>
+</div>
+```
+
+### Events on `oui-step-form`
+
+**Note**: If you want to access the form inside `on-submit` callback, you need to use the `form` variable as below.
+
+```html:preview
+<oui-stepper>
+    <oui-step-form
+        name="eventsStepForm"
+        header="Events on oui-step-form"
+        on-focus="$ctrl.onFocus('Dolor sit amet !')"
+        on-submit="$ctrl.onSubmit(form)">
+        <oui-field label="Lorem ipsum" size="xl">
+            <input class="oui-input" name="focusStep"
+                type="text" required
+                ng-model="$ctrl.onfocusModel">
+        </oui-field>
+    </oui-step-form>
+</oui-stepper>
+<div class="oui-doc-preview-only">
+    <p>On Submit</p>
+    <pre>{{$ctrl.onSubmitForm | json}}</pre>
+</div>
+```
+
+### Custom validation
+
+**Note**: The value of the custom validation attribute doesn't override the angular form validation `form.$valid`. Both are needed to be `true` to enable the submission.
+
+```html:preview
+<div class="oui-doc-preview-only">
+    <p>
+        <button class="oui-button" type="text"
+            ng-class="{
+                'oui-button_primary': $ctrl.isValid,
+                'oui-button_secondary': !$ctrl.isValid
+            }"
+            ng-click="$ctrl.isValid = !$ctrl.isValid">
+            Toggle validation
+        </button>
+    </p>
+</div>
+<oui-stepper>
+    <oui-step-form
+        header="Custom validation"
+        valid="$ctrl.isValid"
+        on-submit="$ctrl.onCustomSubmit()">
+        <oui-field label="Lorem ipsum" size="xl">
+            <input class="oui-input" name="customValidationInput"
+                type="text" autocomplete="off" required
+                ng-model="$ctrl.skippableInput">
+        </oui-field>
+    </oui-step-form>
+</oui-stepper>
+<div class="oui-doc-preview-only">
+    <p><strong>onSubmit count:</strong> {{$ctrl.customSubmitCount}}</p>
+</div>
+```
+
+### Linear stepper
+
+```html:preview
+<oui-stepper linear>
```

Comment:
It could be interesting to have all form elements in this example to validate the behavior
radio-group, numeric, calendar, select-picker..

Reply:
Yes, good idea :)